src/test/ProposalPayloadV2E2E.t.sol

```diff
@@ -25,10 +23,10 @@ contract ProposalPayloadV2E2ETest is ProtocolV2TestBase, TestWithExecutor {
     ProposalPayload public proposalPayload;
 
     // Old Strategies
-    IReserveInterestRateStrategyV2 public constant OLD_INTEREST_RATE_STRATEGY_POLYGON =
-        IReserveInterestRateStrategyV2(0x9025C2d672afA29f43cB59b3035CaCfC401F5D62);
-    IReserveInterestRateStrategyV2 public constant OLD_INTEREST_RATE_STRATEGY_ETHEREUM =
-        IReserveInterestRateStrategyV2(0xfC0Eace19AA7498e0f36eF1607D282a8d6debbDd);
+    IDefaultInterestRateStrategy public constant OLD_INTEREST_RATE_STRATEGY_POLYGON =
+        IDefaultInterestRateStrategy(0x9025C2d672afA29f43cB59b3035CaCfC401F5D62);
+    IDefaultInterestRateStrategy public constant OLD_INTEREST_RATE_STRATEGY_ETHEREUM =
+        IDefaultInterestRateStrategy(0xfC0Eace19AA7498e0f36eF1607D282a8d6debbDd);
```

Comment:
Hey Fermin, on re-review we noticed some addresses here are wrong: https://github.com/llama-community/aave-crv-interest-rate-updates/pull/1/commits/ebb45be04d721fc7b6a9f62cc0767c791fad71ee#diff-0c9385162efd70d5aeb8b9f01472fff2662135317fe9c77e6bb797902e7a4b87R29 

You can use them from address book, to avoid similar mistakes:
https://github.com/bgd-labs/aave-address-book/blob/main/src/AaveV2Ethereum.sol#L317

Same on polygon: https://github.com/bgd-labs/aave-address-book/blob/main/src/AaveV2Polygon.sol#L194